python/nano/src/bigdl/nano/tf/keras/inference_utils.py

```diff
@@ -152,10 +173,7 @@ def quantize(self,
             if isinstance(self, KerasONNXRuntimeModel):     # type: ignore
                 onnx_model = self
             else:
-                spec = tf.TensorSpec((self.input_shape), self.dtype)    # type: ignore
-                onnx_model = self.trace(accelerator='onnxruntime',
-                                        input_sample=spec,
-                                        thread_num=thread_num)
+                onnx_model = self.trace(accelerator='onnxruntime', thread_num=thread_num)
```

Comment:
why don't we need `input_spec` here?

Reply:
now it will automatically infer the `input_spec` in`trace`